src/lib.rs

```diff
@@ -80,6 +82,8 @@ impl Default for ThirdPersonCamera {
             gamepad_settings: CustomGamepadSettings::default(),
             cursor_lock_active: true,
             mouse_sensitivity: 1.0,
+            mouse_orbit_on_button_enabled: false,
+            mouse_orbit_button: MouseButton::Right,
```

Comment:
Since `aim_button` is defaulted to the right mouse button, can you make the default for this `MouseButton::Middle`?

Reply:
done